.github/workflows/releaseConfigurator.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       version: ${{ steps.version.outputs.next-version }}
-      image-tag: ${{ steps.image-tag.outputs.tag }}
+      image-tag: ${{ steps.image-tag.outputs.IMAGE_TAG }}
```

Comment:
I am confused by syntax:
 - next-version is lowercase with a dash and then used like that
 - IMAGE_TAG is uppdercase with a underscore but later on used as lowercase with a dash